tests/regression/98-bitwise-operations/06-bitfield.c

```diff
@@ -0,0 +1,34 @@
+// PARAM: --enable ana.int.interval
+
+#include <stdio.h>
+#include <goblint.h>
+ 
+struct S
+{
+    int a: 3;
+    unsigned int b : 3;  
+};
+ 
+int main(void)
+{
+    int x;
+    struct S s;
+
+    if (x < -30 || x > 30) {
+        x = 0;
+    }
+    
+    s.a = 1; // NOWARN
+    s.b = 1; // NOWARN
+
+    __goblint_check(s.a <= 7);
+    __goblint_check(s.a >= -4);
+    s.a = 8; // WARN
+    s.a = -5; // WARN
+
+    __goblint_check(s.b <= 7);
+    __goblint_check(s.b >= 0);
```

Comment:
All of the `__goblint_check`-s should be before the assignments. Currently the checks are checking the assigned values 1, not the uninitialized top values.
So the checks aren't actually checking the right thing and would even pass using Goblint that doesn't support bitfields.